1.8/usage/service-discovery/mesos-dns/service-naming.md

```diff
@@ -3,9 +3,15 @@ post_title: Service Naming
 menu_order: 0
 ---
 
-Mesos-DNS defines the DNS top-level domain `.mesos` for Mesos tasks that are running on DC/OS. Tasks and services are discovered by looking up A and, optionally, SRV records within this Mesos domain.
+Mesos-DNS defines the DNS top-level domain `.mesos` for Mesos tasks that are running on DC/OS. Tasks and services are discovered by looking up A and, optionally, SRV records within this Mesos domain. 
 
-To enumerate all the DNS records that Mesos-DNS will respond to, take a look at the [DNS naming documentation][4].
+- [A Records](#a-records)
+- [SRV Records](#srv-records)
+- [Other Records](#other-records)
+- [Task and Service Naming Conventions](#naming-conventions)
+- [Discovering Service DNS Names](#dns-naming)
+
+To enumerate all the DNS records that Mesos-DNS will respond to, take a look at this.
```

Comment:
Should this be a link? Maybe change "take a look at this" to "consult that <thing to be consulted>"?